### DISCUSSION TITLE
Changing button color

### DISCUSSION THREAD

**Discussion by hyeokhyen** [greenelab/lab-website-template#261] [Q&A] [answered]
### Checks

- [X] I have searched **[the docs](https://greene-lab.gitbook.io/lab-website-template-docs)**, [existing issues](https://github.com/greenelab/lab-website-template/issues), and [existing discussions](https://github.com/greenelab/lab-website-template/discussions) for answers first.

### Link to your website repo

https://github.com/KwonVitalLab/KwonVitalLab.github.io

### Version of Lab Website Template you are using

1.1.0

### Description

Could we change the color of each button? currently, all buttons are blue. I will appreciate your answer. Thank you!

**Comment by vincerubinetti** [comment, marked as answer]
This is controlled by the variables in `_theme.scss`.

See https://greene-lab.gitbook.io/lab-website-template-docs/basics/customize-your-theme
